projects/bitcoin-core/build.sh

```diff
@@ -15,7 +15,8 @@
 #
 ################################################################################
 
-bash $SRC/build_cryptofuzz.sh
+chmod +x $SRC/build_cryptofuzz.sh
```

Comment:
As an alternative, would it be possible to set the +x bit and then commit the file in git with `git add`?

Reply:
Done